master/src/main/scala/org/apache/celeborn/service/deploy/master/Master.scala

```diff
@@ -658,6 +665,24 @@ private[celeborn] class Master(
     })
   }
 
+  private def cleanExpiredAppDirsOnHDFS(): Unit = {
+    val activeAppIds = statusSystem.getActiveAppIds
+    if (hadoopFs == null) {
+      hadoopFs = CelebornHadoopUtils.getHadoopFS(conf)
+    }
+    val hdfsWorkPath = new Path(conf.hdfsDir, conf.workerWorkingDir)
+    if (hadoopFs.exists(hdfsWorkPath)) {
+      val iter = hadoopFs.listStatusIterator(hdfsWorkPath)
+      while (iter.hasNext) {
+        val fileStatus = iter.next()
+        if (!activeAppIds.contains(fileStatus.getPath.getName)) {
+          logDebug(s"Clean hdfs dir ${fileStatus.getPath.toString}")
```

Comment:
We should print the delete dir message
```suggestion
          logInfo(s"Clean HDFS dir ${fileStatus.getPath.toString}")
```